lib/recurly/purchase.rb

```diff
@@ -189,7 +193,6 @@ def pending!(purchase)
 
       # Allows the merchant to cancel an authorization.
       #
-      # @param transaction_uuid [String] The uuid for the transaction representing the authorization. Can typically be found at invoice_collection.charge_invoice.transactions.first.uuid.
```

Comment:
I think this was removed by accident.

Reply:
@jorge and/or @naspthorp mind putting this line back in?

Reply:
Whoops!  Fixed.